src/components/breadcrumb/breadcrumb.stories.js

```diff
@@ -0,0 +1,29 @@
+import { html } from 'lit-html';
+import { docsIntro, wrapWidth } from '../../../.storybook/utils.js';
+import '.';
+
+export default {
+  title: 'custom-elements/vl-breadcrumb',
+  parameters: {
+    controls: { hideNoControlsWarning: true },
+    docs: {
+      description: {
+        component: docsIntro({
+          root: 'breadcrumb',
+          stylesheets: ['breadcrumb'],
+          intro:
+            'Use breadcrumbs to show the location of the current page within a navigational hierarchy. Breadcrumbs get arrow separators automatically via CSS.',
+        }),
+      },
+    },
+  },
+};
+
+export const Default = () => html`<div style="max-width: ${wrapWidth}">
```

Comment:
Ik zou enkel een wrapper rond de story zetten als het nodig is, niet standaard. In dit geval heeft het geen meerwaarde.